core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -337,34 +338,35 @@ public static void setApplicationName(String applicationName) {
   // TODO(berezv): separate creating socket and performing connection to make it easier to test
   @VisibleForTesting
   Socket createSslSocket(ConnectionConfig config) throws IOException, InterruptedException {
-    CloudSqlInstance instance = getCloudSqlInstance(config);
+    DefaultConnectionInfoCache connectionInfoCache = getConnectionInfoCache(config);
 
     try {
-      SSLSocket socket = instance.createSslSocket(this.refreshTimeoutMs);
+      SSLSocket socket = connectionInfoCache.createSslSocket(this.refreshTimeoutMs);
 
       // TODO(kvg): Support all socket related options listed here:
       // https://dev.mysql.com/doc/connector-j/en/connector-j-reference-configuration-properties.html
       socket.setKeepAlive(true);
       socket.setTcpNoDelay(true);
 
-      String instanceIp = instance.getPreferredIp(config.getIpTypes(), refreshTimeoutMs);
+      String instanceIp = connectionInfoCache.getPreferredIp(config.getIpTypes(), refreshTimeoutMs);
 
       socket.connect(new InetSocketAddress(instanceIp, serverProxyPort));
       socket.startHandshake();
 
       return socket;
     } catch (Exception ex) {
       // TODO(kvg): Let user know about the rate limit
-      instance.forceRefresh();
+      connectionInfoCache.forceRefresh();
       throw ex;
     }
   }
 
-  CloudSqlInstance getCloudSqlInstance(ConnectionConfig config) {
-    return instances.computeIfAbsent(config.getCloudSqlInstance(), k -> apiFetcher(config));
+  DefaultConnectionInfoCache getConnectionInfoCache(ConnectionConfig config) {
+    return connectionInfoCaches.computeIfAbsent(
+        config.getCloudSqlInstance(), k -> apiFetcher(config));
   }
 
-  private CloudSqlInstance apiFetcher(ConnectionConfig config) {
+  private DefaultConnectionInfoCache apiFetcher(ConnectionConfig config) {
```

Comment:
please rename this function to a name more related to `DefaultConnectionInfoCache`

Reply:
Fixed.